deployment/cdk/opensearch-service-migration/dp_pipeline_template.yaml

```diff
@@ -1,18 +1,36 @@
+# Name of the Data Prepper pipeline
 historical-data-migration:
+  # Source cluster configuration
   source:
     opensearch:
+      # DO NOT CHANGE the hosts value - this will be updated by the CDK
       hosts:
       - <SOURCE_CLUSTER_HOST>
+      # Uncomment the following line to disable TLS
+      #insecure: true
+      # Example configuration on how to disable authentication (default: false)
       disable_authentication: true
       indices:
+        # Indices to exclude - exclude system indices by default
         exclude:
         - index_name_regex: \.*
+  # Target cluster configuration
   sink:
   - opensearch:
-      bulk_size: 10
+      # DO NOT CHANGE the hosts value - this will be updated by the CDK
```

Comment:
Same as comment above: I would add a note that they would need to change these only if running locally (in Docker)

Reply:
After the [most recent updates to the Fetch Migration CDK](https://github.com/opensearch-project/opensearch-migrations/commit/353ab1168cee47338d4b51f5bf65acc8e2f5213f), the `hosts` value in the `sink` will be replaced by an [inline target host value](https://github.com/opensearch-project/opensearch-migrations/blob/44978733f3bc0cb3a61981daf4c078ebdf33b4e2/deployment/cdk/opensearch-service-migration/lib/fetch-migration-stack.ts#L84) through the CDK, so changing this value will not affect that logic nor reflect in the final setup. I've updated the comment here to:

```
# Note - CDK code will replace this value with the target cluster endpoint
```

This is in contrast to the `hosts` value in `source` where the string value matters because the CDK [performs a string substitution](https://github.com/opensearch-project/opensearch-migrations/blob/44978733f3bc0cb3a61981daf4c078ebdf33b4e2/deployment/cdk/opensearch-service-migration/lib/fetch-migration-stack.ts#L71) to inject the source cluster endpoint.